modules/doc/content/newsletter/2025/2025_03.md

```diff
@@ -1,14 +1,109 @@
+|                   Object                    | Time | Stepinterval Count | Total Count |
+-----------------------------------------------------------------------------------------
+| NonsafeMaterial : Solution invalid warning! | 0-1  |                  0 |           0 |
+| NonsafeMaterial : Solution invalid warning! | 1-2  |                 48 |          48 |
+| NonsafeMaterial : Solution invalid warning! | 2-3  |                 48 |          96 |
+-----------------------------------------------------------------------------------------
+```
+
+### New Automatic Differentiation (AD) Curl Methods
+
+Automatic differentiation capability has been extended to curl operations in the form of `adCurlSln()`.
+In addition, coupling the curl of vector field variables can be called by using `adCoupledCurl` and
+new base classes labeled `KernelCurl`, `ADKernelCurl`, and `GenericKernelCurl`  have been created to
+facilitate the templating of future curl based kernel objects. When an object inherence from `GenericKernelCurl`,
+it gains access to the following new variables:
+
+- `_curl_u`: Holds the solution curl at the current quadrature points
+- `_curl_test`: Curl of the test function
+- `_curl_phi`: Curl of the shape function
+
+### New ActionComponents interfaces
+
+Three interfaces were added to be used on [ActionComponents](syntax/ActionComponents/index.md) as need:
+
+- [ComponentBoundaryConditionInterface.md] to define boundary conditions on the components that get interpreted by the [Physics](syntax/Physics/index.md)
+- [ComponentInitialConditionInterface.md] to define initial conditions on the components that get interpreted by the [Physics](syntax/Physics/index.md)
+- [ComponentMaterialPropertyInterface.md] to define generic material properties directly on the components on which we plan to use them
+
+
+### Restart capability improvements
+
+Simulations using p-refinement can now be restarted using a [Checkpoint.md].
+MOOSE will also no longer attempt to repartition after loading a checkpoint file
+
+### Framework-level, user-facing API for loading and evaluating torchscript objects
+
+The ability to load torch script models into MOOSE facilitates the use of UQ/ML using libtorch directly in simulations. We also want to ensure that these models are loadable on the fly at different execute_on options.
+
+The [TorchScriptUserObject.md] stores the torchscript object.
+
+
 ## MOOSE Modules Changes
 
+### New Subchannel module
+
+After it was cleared for release as open-source software, the Subchannel application was merged in MOOSE as the Subchannel module (SCM). SCM is a Subchannel Module for performing reactor core, single-phase thermal-hydraulic subchannel simulations, for bare pin, square lattice bundles or wire-wrapped/bare pin, triangular lattice bundles. It can model water cooled or metal cooled (lead, sodium, LBE) bundles.
+
+The subchannel documentation may be found at this [link](subchannel/doc/content/index.md).
```

Comment:
```suggestion
The formerly closed-source Subchannel application was merged as the Subchannel module (SCM). SCM is a Subchannel Module for performing reactor core, single-phase thermal-hydraulic subchannel simulations, for bare pin, square lattice bundles or wire-wrapped/bare pin, triangular lattice bundles. It can model water cooled or metal cooled (lead, sodium, LBE) bundles.

The subchannel documentation may be found at this link
```